### PR TITLE
add reply logic

### PR DESCRIPTION
**Pull request by prokhorind**

### CODE REVIEW

File: src/main/java/com/mattermost/integration/figma/api/comment/service/PostCommentServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    private static final String BASE_URL = "https://api.figma.com/v1/files/%s/comments";
+
+    @Autowired
+    private RestTemplate restTemplate;
+
+    @Override
+    public void postComment(String fileId, String replyCommentId, String message, String token) {
+        String url = String.format(BASE_URL, fileId);
+
+        var postCommentRequestDTO = new PostCommentRequestDTO();
```
Comment (OstapMelnychuk) [inline src/main/java/com/mattermost/integration/figma/api/comment/service/PostCommentServiceImpl.java:23]: Please use the appropriate data type here (PostCommentRequestDTO)

File: src/main/java/com/mattermost/integration/figma/api/comment/service/PostCommentServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        postCommentRequestDTO.setCommentId(replyCommentId);
+        postCommentRequestDTO.setMessage(message);
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.set("Authorization", String.format("Bearer %s", token));
+
+        var request = new HttpEntity(postCommentRequestDTO ,headers);
```
Comment (OstapMelnychuk) [inline src/main/java/com/mattermost/integration/figma/api/comment/service/PostCommentServiceImpl.java:30]: Please use the appropriate data type here (HttpEntity<PostCommentRequestDTO>)

File: src/main/java/com/mattermost/integration/figma/api/comment/service/PostCommentServiceImpl.java (hunk 1 above)
Comment (prokhorind) [reply src/main/java/com/mattermost/integration/figma/api/comment/service/PostCommentServiceImpl.java]: fixed

File: src/main/java/com/mattermost/integration/figma/api/comment/service/PostCommentServiceImpl.java (hunk 2 above)
Comment (prokhorind) [reply src/main/java/com/mattermost/integration/figma/api/comment/service/PostCommentServiceImpl.java]: fixed

**Pull request merged**